tests/test_enterprise_learner_portal/api/test_views.py

```diff
@@ -16,8 +17,14 @@
 from enterprise.utils import NotConnectedToOpenEdX
 from test_utils import factories
 
+FAKE_SERIALIZED_DATA = [
```

Comment:
DRY nit: each of these course enrollment data objects can be broken out into their own variables (e.g. `ACTIVE_ENROLLMENT`, `INACTIVE_ENROLLMENT`) that way if they need to be expanded upon it can be done in one place. Also nitty - I feel like this variable name can be more descriptive but I'm not going to die on that hill 😂 up to you